docs/tutorials/otio-timeline-structure.md

```diff
@@ -5,6 +5,26 @@ clips, gaps, and transitions. This document is meant to clarify how these
 objects nest within each other, and how they work together to represent an
 audio/video timeline.
 
+## Rendering
+
+Rendering of the image tracks in a timeline is done in painter order. The layers
+in a stack are iterated from the bottom (the first entry in the stack) towards
+the top (the final entry in the stack). Images in a stack overlay lower images
+with no blending.
```

Comment:
As @reinecke mentioned in the Slack thread, the current documentation recommends the tracks be alpha composited:
https://github.com/AcademySoftwareFoundation/OpenTimelineIO/wiki/OpenTimelineIO-Application-Integrator's-Guide#stack

That seems like it would be preferable for things like information overlays (logos, shot info, etc.)?